### ISSUE TITLE
TO-DO: Seperate fetch logic from icon logic.

### ISSUE THREAD

**Issue by Delubear** [Delubear/GlucoseTray#13]
Create a project that contains all the fetch information.  This can then be moved into a nuget package that others can use to fetch their results from either Dexcom or Nightscout.

**Comment by rwkarg** [comment]
Looks like there's a single Dexcom.Fetch right now that has both Dexcom Share and Nightscout functionality. Is this in the process of being pulled in to two separate packages?

I started last week pulling the fetch services out in to a netstandard package but it looks like you've already started work on that before I got around to committing:
https://github.com/rwkarg/GlucoseTray/commit/49025d8e462ce298289cda930570c4b911b1c585
(turns out to be an ugly merge since it also includes moving the projects out of the root file directory. Only real change is in the GlucoseTray.Services project)
